app/src/components/v-table/table-header.vue

```diff
@@ -441,6 +441,8 @@ function toggleManualSort() {
 	background-color: var(--foreground-subdued);
 	display: inline-block;
 	border-radius: 50%;
+	border: var(--background-page) 6px solid;
```

Comment:
Would be a bit cleaner IMO
```suggestion
	border: transparent 6px solid;
```

Reply:
No that doesnt work because then you'll see the background color underneath :angel:

Reply:
Of course, I didn't notice the dot is composed out of background-color / radius 😅👍